### PR TITLE
[ENH] Ontology widget documentation

### PR DESCRIPTION
**Pull request by PrimozGodec**
##### Issue

The ontology widget is missing the documentation

##### Description of changes
Add documentation for the ontology widget

##### Includes
- [ ] Code changes
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by ajdapretnar** [comment]
I was planning to do this, but not before Ontology is a bit more stable. For instance, #832 and #824.

File: doc/widgets/ontology.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+- Words: A table of words.
+
+**Outputs**
+
+- Words: A table of words.
+
+The ontology widget enables various operations with ontologies:
+- Generate ontology from words on the input
+- Load existing ontology from a file or URL
```
Comment (ajdapretnar) [inline doc/widgets/ontology.md:16]: Add which formats are supported.

**Comment by PrimozGodec** [comment]
> I was planning to do this, but not before Ontology is a bit more stable. For instance, #832 and #824.

I started to work on it since on MJU they said that they need it to finish the project. 
Regarding the issues:
#832 - this one is a really difficult one since it is caused by some unexpected behaviour by PyQT. I will continue working on it but haven't had enough time yet.
#824 - is also waiting because of me not having enough time

File: doc/widgets/ontology.md (hunk 1 above)
Comment (PrimozGodec) [reply doc/widgets/ontology.md]: Done

**Comment by ajdapretnar** [comment]
@PrimozGodec I am sorry I didn't check this yesterday, but you need to add `ontology.md` to `index.rst` and re-create `widgets.json`.

**Pull request merged**